app/controllers/api/health.php

```diff
@@ -268,7 +267,7 @@
                 'status' => (@$antiVirus->ping()) ? 'online' : 'offline',
                 'version' => @$antiVirus->version(),
             ]);
-        } catch( RuntimeException $e) {
+        } catch( \RuntimeException $e) {
```

Comment:
We should just use the more common PHP `Exception` or `Throwable` to catch any possible error.